packages/x-data-grid/src/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -1104,3 +1104,7 @@ function bufferForDirection(
       throw new Error('unreachable');
   }
 }
+
+function roundToSubPixel(value: number) {
+  return Math.round(value * 10) / 10;
+}
```

Comment:
Why is 1 decimal place a better value?

Reply:
Some browsers support subpixel positioning, but I saw some instances in the docs where width was 242.32px, which triggers unnecessary updates. Happy to change it to full pixels if that makes more sense.